diff -urN a/cmake_modules/FindLZ4.cmake b/cmake_modules/FindLZ4.cmake
--- a/cmake_modules/FindLZ4.cmake	2022-04-10 00:50:16.000000000 +0300
+++ b/cmake_modules/FindLZ4.cmake	2022-05-17 12:30:19.747281995 +0300
@@ -28,7 +28,7 @@
   NO_DEFAULT_PATH
   PATH_SUFFIXES "include")
 
-find_library (LZ4_LIBRARY NAMES lz4 HINTS
+find_library (LZ4_LIBRARY NAMES liblz4 HINTS
   ${_lz4_path}
   PATH_SUFFIXES "lib" "lib64")
 
diff -urN a/cmake_modules/FindProtobuf.cmake b/cmake_modules/FindProtobuf.cmake
--- a/cmake_modules/FindProtobuf.cmake	2022-04-10 00:50:16.000000000 +0300
+++ b/cmake_modules/FindProtobuf.cmake	2022-05-18 09:35:44.997608503 +0300
@@ -37,7 +37,7 @@
   NO_DEFAULT_PATH
   PATH_SUFFIXES "include")
 
-find_library (PROTOBUF_LIBRARY NAMES protobuf HINTS
+find_library (PROTOBUF_LIBRARY NAMES libprotobuf HINTS
   ${_protobuf_path}
   PATH_SUFFIXES "lib")
 
