From 242d58eb0ddf024a3dfe791c5993fdfdb99d17dd Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Fri, 15 Jun 2018 15:01:07 -0500
Subject: [PATCH] remove Adaptor.hh include in Exceptions.hh for portability

---
 c++/src/Exceptions.hh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/c++/src/Exceptions.hh b/c++/src/Exceptions.hh
index 4706085..5d1e261 100644
--- a/c++/src/Exceptions.hh
+++ b/c++/src/Exceptions.hh
@@ -19,8 +19,6 @@
 #ifndef ORC_EXCEPTIONS_HH
 #define ORC_EXCEPTIONS_HH
 
-#include "Adaptor.hh"
-
 #include <stdexcept>
 #include <string>
 
-- 
2.11.1

